### PR TITLE
pypechain: make it drier

### PR DESCRIPTION
**Pull request by wakamex**
- replace `get_input_names` and `get_input_names_and_values` with single `get` function
- replace `stringify_parameters` with `get` which renames anonymous functions to input1, output2, etc.
- simplify conditional statement in abi.py
- lint

### CODE REVIEW

**Comment by wakamex** [comment]
no changes on ERC20Contract.py

File: lib/pypechain/pypechain/run_pypechain.py [hunk 1]
```diff
@@ -142,96 +140,24 @@ def render_types_file(contract_name: str, types_template: Template, abi_file_pat
...
-    stringified_function_parameters: list[str] = []
-    for _input in function.get("inputs", []):
-        if name := get_param_name(_input):
-            python_type = solidity_to_python_type(_input.get("type", "unknown"))
-        else:
-            raise ValueError("Solidity function parameter name cannot be None")
-        stringified_function_parameters.append(f"{avoid_python_keywords(name)}: {python_type}")
-    return stringified_function_parameters
+def get(function: ABIFunction, param_type, include_types: bool = True) -> list[str] | dict[str, str]:
+    """Returns function inputs or outputs, optionally including types."""
```
Comment (dpaiton) [inline lib/pypechain/pypechain/run_pypechain.py:144]: Please add more complete doc strings.

Comment (wakamex) [reply lib/pypechain/pypechain/run_pypechain.py]: updated

**Comment by wakamex** [comment]
needs a rebase

**Comment by dpaiton** [comment]
closing this -- pypechain lives in its own repo now

**Pull request closed without merging**